test/e2e/README.md

```diff
@@ -24,11 +24,25 @@ knative-gcp should be added under [knative-gcp e2e test lib](lib).
 
 ## Running E2E Tests on an existing cluster
 
-To run [the e2e tests](../e2e) with `go test` command, you need to have a
-running environment that meets
-[the e2e test environment requirements](#environment-requirements), and you need
-to specify the build tag `e2e`.
+### Prerequisites
 
+1. A running Kubernetes cluster with [knative-gcp](../../docs/install/install-knative-gcp.md) installed and configured.
+2. [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md) installed.
+3. [Broker with Pub/Sub Channel](../../docs/install/install-broker-with-pubsub-channel.md) installed.
+4. A docker repo containing [the test images](#test-images). Remember to specify the build tag `e2e`.
+5. (Optional) Note that if you plan on running metrics-related E2E tests using the StackDriver
```

Comment:
Anyone tried to run E2E tests with `TestChannelTracing`.
When I tried to run `E2E_PROJECT_ID=$PROJECT_ID  go test --tags=e2e ./test/e2e/... -run TestChannelTracing`, it failed with `Error retrieving Zipkin pod details: no zipkin Pod found on the cluster. Ensure monitoring is switched on for your Knative Setup zipkin.go:35: Unable to set up Zipkin for tracking`.
I already give the service account the `Monitoring Editor` role. Do I need to do other set up besides the Monitoring Editor role? In addition, the service account should be the service account of Data Plane, right?

Reply:
@ian-mi  Do you know how to set up this Zipkin?